eksconfig/add-on-managed-node-groups.go

```diff
@@ -139,10 +139,33 @@ type MNG struct {
 	// Logs maps each instance ID to a list of log file paths fetched via SSH access.
 	Logs map[string][]string `json:"logs" read-only:"true"`
 
+	// ConfigUpdate configures MNG config update.
+	ConfigUpdates []*MNGScaleUpdate `json:"scale-update,omitempty"`
```

Comment:
Also, can we rename this to `ScaleUpdates`? And `json:"scale-updates,omitempty"`